lit-next.code-workspace

```diff
@@ -62,7 +62,10 @@
     "debug.javascript.warnOnLongPrediction": false,
     // Regardless of the user's usual formatter, files
     // in the Lit repo must be formatted using prettier.
-    "editor.defaultFormatter": "esbenp.prettier-vscode"
+    "editor.defaultFormatter": "esbenp.prettier-vscode",
+    "files.exclude": {
```

Comment:
Doesn't it already ignore everything in gitignore?

Reply:
Not completely, e.g. wireit files fill the cmd+P menu